docs/modules/ROOT/pages/how-to/remove-default-networkpolicies.adoc

```diff
@@ -0,0 +1,36 @@
+= Remove Default NetworkPolicies
+
+In every namespace 2 `NetworkPolicies` are created and maintained by {product}:
+
+. `allow-from-other-namespaces`: This policy allows the Router and other system components to connect to the pods.
+. `allow-from-same-namespace`: This policy allows connections between pods in the same namespace.
+
+NOTE: Changes to these policies are reverted to their original state.
+
+If you have the need to customize the default policies, you can remove them and provide your own policies.
+To customize the policies, configure the namespace labels as following:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: ""
+    openshift.io/display-name: My cool project
+    openshift.io/requester: my-username
+  labels:
+    appuio.io/organization: my-company
+    kubernetes.io/metadata.name: my-namespace
+    network-policies.syn.tools/no-defaults: 'true' <1>
+    network-policies.syn.tools/purge-defaults: 'true' <2>
+  name: my-namespace
+----
+<1> Set this label to prevent reverting the default network policies in case you edit them.
```

Comment:
```suggestion
<1> Add this label to prevent {product} from reverting changes to the default network policies.
Note that {product} won't recreate the default network policies (for example if they are accidentally deleted) if this label is set to `true`.
Also note that {product} won't create the default network policies if this label is set to `true` when the namespace is created.
```